src/granite/association_collection.cr

```diff
@@ -11,26 +11,23 @@ class Granite::AssociationCollection(Owner, Target)
     )
   end
 
-  def find_by(field : String | Symbol, value)
+  def find_by(**args)
```

Comment:
I like the new syntax, should the old overload stay around too?

Reply:
That was the original plan, but i didn't like the inconsistency between the two.  IMO id just keep the one and people can take a few min to change their statements to new format, which is better as well, as they would have to anyway to use the multiple criteria feature anyway.